bundles/org.openhab.binding.paradoxalarm/README.md

```diff
@@ -93,15 +93,15 @@ Currently binding supports the following panels: EVO192, EVO48(not tested), EVO9
 | forceReady               | Switch  | Partition is in state Force Ready                                                             |
 | bypassReady              | Switch  | Partition is in state Bypass Ready                                                            |
 | inhibitReady             | Switch  | Partition is in state Inhibit Ready                                                           |
-| allZonesClosed           | Contact | All zones in partition are currently closed                                                   |
-| command                  | String  | Command to be send to partition. Can be (ARM, DISARM, FORCE_ARM, INSTANT_ARM, STAY_ARM, BEEP) |
+| allZonesClosed           | Contact | All zones in partition are currently closed.                                                  |
+| command                  | String  | Command to be sent to partition. Can be (ARM, DISARM, FORCE_ARM, INSTANT_ARM, STAY_ARM, BEEP). |
```

Comment:
It would be nice if this table could be reformatted as it was previously (almost) nicely aligned.

Reply:
In between your review and publishing it, i changed it already ;-)